deno.json

```diff
@@ -28,5 +29,6 @@
   ],
   "fmt": {
     "indentWidth": 4
-  }
-}
\ No newline at end of file
+  },
+  "nodeModulesDir": "auto"
```

Comment:
this is just to get Canvas working - if you know a better way that'd be great. I'm relatively new to deno.

Reply:
I wonder if we could get away with overriding the canvas api somewhere by one of these:

- https://github.com/DjDeveloperr/deno-canvas
- https://docs.deno.com/api/web/canvas

Here someone explained that we managed to override some api in order to make canvas working: https://stackoverflow.com/a/46571161

And someone managed to "mock" canvas in jsdom: https://github.com/jsdom/jsdom/issues/1782#issuecomment-337656878

Reply:
I've removed the canvas stuff - the error still pops up when the token refreshes/server starts, but it seems to be harmless. The error gets swallowed by botguard and then it just carries on as if it never happened. 

I imagine we can just ignore the issue if you're ok with the error message showing in the log. It also works with a compiled version. I've reverted the addition of the canvas package.